pkg/meta/annotations.go

```diff
@@ -300,6 +300,11 @@ const (
 	OnlineUpgradeArchiveBeforeReplicationAnnotation = "vertica.com/online-upgrade-archive-before-replication"
 
 	SaveRestorePointAnnotation = "vertica.com/save-restore-point-on-upgrade"
+	// Allow users to skip the creation of a VerticaRestorePointsQuery object after saving
+	// a restore point. When this is false, after saving a restore point, the operator
+	// will create a VerticaRestorePointsQuery in order to expose the restore point info
+	// the user.
+	SkipVRPQCreationAnnotation = "vertica.com/skip-restore-points-query-creation"
```

Comment:
Why do we need this annotation? We should always create the VRep CR to collect the restore point's info for the user.